services/web/server/tests/unit/with_dbs/02/test_projects_comments_handlers.py

```diff
@@ -0,0 +1,190 @@
+        json={"contents": updated_comment},
+    )
+    data = await resp.json()
+    assert resp.status == 200
+    assert data["data"]["contents"] == updated_comment
+
+    # Now we will get the second comment
+    resp = await client.get(base_url / f"{second_comment_id}")
+    data = await resp.json()
+    assert resp.status == 200
+    assert data["data"]["contents"] == updated_comment
+
+    # Now we will delete the second comment
+    resp = await client.delete(base_url / f"{second_comment_id}")
+    data = await resp.json()
+    assert resp.status == 204
+
+    # Now we will list all comments for the project
+    resp = await client.get(base_url)
+    data = await resp.json()
+    assert resp.status == 200
+    assert len(data["data"]) == 1
+
+    # Now we will log as a different user
+    async with LoggedUser(client) as new_logged_user:
+        # As this user does not have access to the project, they should get 404
+        resp = await client.get(base_url)
+        data = await resp.json()
+        assert resp.status == 404
+
+        resp = await client.get(base_url / f"{first_comment_id}")
+        data = await resp.json()
+        assert resp.status == 404
+
+        # Now we will share the project with the new user
+        with postgres_db.connect() as con:
+            result = con.execute(
+                projects.update()
+                .values(
+                    **{
+                        "access_rights": {
+                            str(logged_user["primary_gid"]): {
+                                "read": True,
+                                "write": True,
+                                "delete": True,
+                            },
+                            str(new_logged_user["primary_gid"]): {
+                                "read": True,
+                                "write": True,
+                                "delete": True,
+                            },
+                        }
+                    }
+                )
+                .where(projects.c.uuid == user_project["uuid"])
+            )
+
+        # Now the user should have access to the project now
+        # New user will add comment
+        resp = await client.post(
+            base_url,
+            json={"contents": "My first comment as a new user"},
+        )
+        assert resp.status == 201
+        data = await resp.json()
+        new_user_comment_id = data["data"]["comment_id"]
+
+        # New user will modify the comment
+        updated_comment = "Updated My first comment as a new user"
+        resp = await client.put(
+            base_url / f"{new_user_comment_id}",
+            json={"contents": updated_comment},
+        )
+        data = await resp.json()
+        assert resp.status == 200
+        assert data["data"]["contents"] == updated_comment
+
+        # New user will list all comments
+        resp = await client.get(base_url)
+        data = await resp.json()
```

Comment:
MINOR: check ``assert_status`` helper in https://github.com/ITISFoundation/osparc-simcore/blob/cf422bb2b61b8ae738eaad79f34f1de7c2bce995/packages/pytest-simcore/src/pytest_simcore/helpers/utils_assert.py

it simplifies tests

Reply:
done :+1: